README.md

```diff
@@ -1,44 +1,69 @@
-GeoMesa NiFi Bundle
-===================
+# GeoMesa NiFi
 
-NiFi manages large batches and streams of files and data.
-GeoMesa-NiFi allows you to ingest data into GeoMesa straight from NiFi by
-leveraging custom processors.
+GeoMesa-NiFi allows you to ingest data into GeoMesa using the NiFi dataflow framework. Currently, the following datastores are supported:
 
-Installation
-------------
+* Accumulo
+* HBase
+* Kafka 09
+* Kafka 10
 
-### Install the Processors
+# Building from Source
 
-Pick a reasonable directory on your machine, and run:
+To build, simply clone and build with maven:
 
 ```bash
-$ git clone https://github.com/geomesa/geomesa-nifi.git
-$ cd geomesa-nifi
+git clone git@github.com:geomesa/geomesa-nifi.git
+cd geomesa-nifi
+mvn clean install
 ```
 
-To build, run
+# Installation
+
+To install the GeoMesa processors you will need to copy the appropriate nar file into the ``lib`` directory of your NiFi installation. There are
+three nar files that are built currently:
+
+Nar                                | Datastores
+---                                | ---
+geomesa-nifi-nar-$VERSION.nar      | Accumulo, HBase
+geomesa-nifi-kafka-09-$VERSION.nar | Kafka 09
+geomesa-nifi-kafka-10-$VERSION.nar | Kafka 10
+
+For example, to install the Acccumulo or HBase nar after building from
+source:
 
 ```bash
-$ mvn clean install
-```
+cp geomesa-nifi/geomesa-nifi-nar/target/geomesa-nifi-nar-$VERSION.nar $NIFI_HOME/lib/
+````
+
+# SFTs and Converters
+
+GeoMesa NiFi nar files package a set of predefined SimpleFeatureType schema definitions and GeoMesa Converter definitions for popular data
+sources including:
+
+* Twitter
+* GDelt
+* OpenStreetMaps
+
+The full list of converters can be found in the GeoMesa source:
+
+https://github.com/locationtech/geomesa/tree/master/geomesa-tools/conf/sfts
+
+## Defining custom SFTs and Converters
 
-To install the GeoMesa processors you will need to copy the geomesa-nifi-nar file from
-``geomesa-nifi/geomesa-nifi-nar/target/geomesa-nifi-nar-$VERSION.nar`` into the ``lib/`` directory of your NiFi installation.
+There are two ways of providing custom SFTs and converters:
 
-### Install the SFTs and Converters
+* Providing a ``reference.conf`` file via a jar in the ``lib`` dir
```

Comment:
Both

It can be in the geomesa-nifi nar but not another nar. Due to the way the class loaders work inside nifi if it is in a different nar this one won't be able to see it. (I may have spent a day working this one out)